common-arcana.lic

```diff
@@ -49,6 +49,9 @@ module DRCA
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
       return prepare?(abbrev, mana, symbiosis, command)
+    when 'You should stop playing before you do that'
+      fput('stop play')
```

Comment:
Fixing bput looks to be a much better solution.  I've updated the PR to be a bput change instead